Source/Classes/NantesLabel.swift

```diff
@@ -634,7 +649,7 @@ public extension NSAttributedString.Key {
         enabledTextCheckingTypes = [.link, .address, .phoneNumber]
     }
 
-    private func characterIndex(at point: CGPoint) -> Int {
+    private func characterIndex(at point: CGPoint, in attributedText: NSAttributedString, framesetter: CTFramesetter) -> Int {
```

Comment:
`framesetter` doesn't seem to be used in this function?

Reply:
`let frame = CTFramesetterCreateFrame(framesetter, CFRange(location: 0, length: attributedText.length), path, nil)` just a little bit down in there. You can't see it in the diff because `framesetter` was there before from self so that line didn't need to change.

Reply:
ah right, apologies

Reply:
👍 all good! Thanks for taking the time to check it out :)